lib/liquid/tokenizer.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Liquid.Tokenizer do
+  @moduledoc """
+  Prepares markup to be parsed
```

Comment:
Can you add more documentation on _how_? For instance: Prepares markup to be parsed by splitting up...

Reply:
Sure, we'll add more documentation in the next commit